compliance/http/src/test/java/org/eclipse/rdf4j/http/server/ProtocolTest.java

```diff
@@ -482,6 +516,62 @@ public void run() {
 		threadPool.shutdownNow();
 	}
 
+	private Set<Namespace> getNamespaces(String location)
+		throws Exception
+	{
+		URL url = new URL(location);
+		HttpURLConnection conn = (HttpURLConnection)url.openConnection();
+		try {
+			conn.setRequestProperty("Accept", "text/csv; charset=utf-8");
+			conn.connect();
+
+			InputStream connIs = conn.getInputStream();
+			try {
+				int responseCode = conn.getResponseCode();
+				if (responseCode != HttpURLConnection.HTTP_OK) {
+					String response = "location " + location + " responded: " + conn.getResponseMessage()
+							+ " (" + responseCode + ")";
+					fail(response);
+				}
+
+				String responseText = IOUtil.readString(connIs);
+				String[] rows = responseText.split("\\r?\\n");
```

Comment:
@ansell Just to be sure. Was your suggestion to replace my code for parsing the CSV response with the use of the opencsv library? At first, I thought that I should use the class `org.eclipse.rdf4j.query.resultio.text.csv.SPARQLResultsCSVParser`. However, now I think that direct use of opencsv could be better.